generator/script/helpers.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Helpers
+ */
+const escapeTest = /[&<>"']/;
+const escapeReplace = /[&<>"']/g;
+const escapeTestNoEncode = /[<>"']|&(?!#?\w+;)/;
+const escapeReplaceNoEncode = /[<>"']|&(?!#?\w+;)/g;
+const escapeReplacements = {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    "'": '&#39;'
+};
+const getEscapeReplacement = (ch) => escapeReplacements[ch];
```

Comment:
prefer to use a lib like https://www.npmjs.com/package/escape-html because you probably not handle all the case of escape ...